packages/fast-components-react-base/src/dialog/dialog.tsx

```diff
@@ -1,10 +1,12 @@
 import { DialogClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
-import { classNames, keyCodeEscape } from "@microsoft/fast-web-utilities";
+import { classNames, keyCodeEscape, keyCodeTab } from "@microsoft/fast-web-utilities";
 import { canUseDOM } from "exenv-es6";
 import React from "react";
 import { DisplayNamePrefix } from "../utilities";
 import { DialogHandledProps, DialogProps, DialogUnhandledProps } from "./dialog.props";
+import { isNil } from "lodash-es";
+import Tabbable from "tabbable";
```

Comment:
Do we have any concerns about this *not* being ES6 and the size delta it could create downstream? @nicholasrice

Reply:
I don't have much concern about this not using ES6 modules - it is a purpose-specific dep that doesn't (at least at this point) appear to have any code that can be tree-shaken.

Bundle size is aways a concern - if we think we can do this with less code then we should.

Reply:
Thoughts on the above @nicholasrice? I'm open to either here, but want to be conscious of bundle size increases on folks.